dream_layer_frontend/src/features/Extras/ExtrasPage.tsx

```
@@ -376,6 +384,61 @@ const ExtrasPage = () => {
+            settings: {
+              model: img.settings?.model || 'unknown',
+              sampler: img.settings?.sampler || 'unknown',
+              steps: img.settings?.steps || 20,
+              cfg_scale: img.settings?.cfg_scale || 7.0,
+              seed: img.settings?.seed || -1,
+              width: img.settings?.width || 512,
+              height: img.settings?.height || 512,
+              ...img.settings
+            }
+          };
+        });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)